model/src/main/java/org/mskcc/cmo/metadb/model/CmoRequestEntity.java

```diff
@@ -55,5 +56,15 @@ public String getRequestJson() {
     public void setRequestJson(String requestJson) {
         this.requestJson = requestJson;
     }
-
+    
+    /**
+     * 
+     * @param sampleManifestEntity
+     */
+    public void addSampleManifest(SampleManifestEntity sampleManifestEntity) {
+        if (sampleManifestList == null) {
+            sampleManifestList = new ArrayList<SampleManifestEntity>();
```

Comment:
You shouldnt need the type in the carrot brackets in java 8+, let's remove to keep consistent with the styling elsewhere in the code